spec/rspec/core/configuration_spec.rb

```diff
@@ -803,6 +796,42 @@ def specify_consistent_ordering_of_files_to_run
       end
     end
 
+    context "with an example id" do
+      it "assigns the file and id as an ids filter" do
+        assign_files_or_directories_to_run "./path/to/a_spec.rb[1:2]"
+        expect(inclusion_filter).to eq(:ids => { "./path/to/a_spec.rb" => ["1:2"] })
```

Comment:
Why `{ "./path/to/a_spec.rb" => ["1:2"] }` instead of `{ "./path/to/a_spec.rb" => [1, 2] }` or `{ "./path/to/a_spec.rb" => ["1", "2"] }`?